shell/common/gin_converters/content_converter.cc

```diff
@@ -188,6 +212,14 @@ v8::Local<v8::Value> Converter<content::PermissionType>::ToV8(
       return StringToV8(isolate, "mediaKeySystem");
     case content::PermissionType::MIDI:
       return StringToV8(isolate, "midi");
+    case content::PermissionType::WAKE_LOCK_SCREEN:
+      return StringToV8(isolate, "screen-wake-lock");
+    case content::PermissionType::SENSORS:
+      return StringToV8(isolate, "sensors");
+    case content::PermissionType::WAKE_LOCK_SYSTEM:
+      return StringToV8(isolate, "system-wake-lock");
+    case content::PermissionType::WINDOW_PLACEMENT:
+      return StringToV8(isolate, "window-placement");
     default:
```

Comment:
If we've truly handled all possible permissions can we remove this `default` case as it should be unreachable?

Reply:
should we remove it? or just replace it with an `UNREACHABLE()` `DCHECK`?

Reply:
If we remove it the compiler will yell at us if we miss a new enum value

Reply:
@codebytere We don't want an unreachable here because there is another switch statement after this one to handle some additional permission types Electron has defined.  @MarshallOfSound As far as default, I thought it was a coding style thing to always include a default on a switch statement.

Reply:
Google's [style guide](https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements) says that:
>If not conditional on an enumerated value, switch statements should always have a default case

But PermissionType has this weird "NUM" type that isn't used: https://chromium.googlesource.com/chromium/src/+/master/content/public/browser/permission_type.h#55

Reply:
Ah... chromium just uses a break on `NUM`:
https://source.chromium.org/chromium/chromium/src/+/master:components/permissions/permission_manager.cc;l=133

I'll update to handle all the defined enum values and remove the default.